CMakeLists.txt

```diff
@@ -276,15 +289,95 @@ else()
     target_compile_definitions(openPMD PUBLIC "-DopenPMD_HAVE_HDF5=0")
 endif()
 
+add_library(openPMD.ADIOS1.Serial SHARED ${IO_ADIOS1_SEQUENTIAL_SOURCE})
+add_library(openPMD.ADIOS1.Parallel SHARED ${IO_ADIOS1_SOURCE})
```

Comment:
Just for better understanding:
Are all 4 combinations

SER PAR
 N   N
 N   Y
 Y   N
 Y   Y

are possible with this change?

Reply:
Yes indeed :dancer: 

Even though an ADIOS install of the form
```
SER PAR
 N   Y
```

is not build by ADIOS (it always ships at least a serial lib).

Reply:
Impressive!
... and that only requires building two seperate library files by compiling with different options and linking them with tinkered symbols.

> its lovely mock magic

:joy:

Reply:
No symbol tinkering, just really aggressive symbols hiding for encapsulation :smiling_imp: